lib/grinder_tools.dart

```diff
@@ -217,9 +225,11 @@ class TestRunner {
   /// `firefox`, `safari`. [concurrency] controls the number of concurrent test
   /// suites run (defaults to 4). [pubServe] is the port of a pub serve instance
   /// serving `test/`.
-  Future testAsync({String name, String plainName, dynamic platformSelector,
-      int concurrency, int pubServe, RunOptions runOptions}) {
+  Future testAsync({dynamic files, String name, String plainName,
+      dynamic platformSelector, int concurrency, int pubServe,
+      RunOptions runOptions}) {
     return _test.runAsync(_buildArgs(
+        files: files,
```

Comment:
Totally agree re: the optional positional parameter; would have preferred to have something like that available. I think the primary use-case will be files however; would prefer to keep it as is.